lib/enum/arrays.rb

```diff
@@ -1,24 +1,32 @@
 class Array
 
   def sum_of_numeric_elements
+    select { |el| el.is_a?(Numeric) }.inject(:+)
   end
 
   def product_of_numeric_elements
+    select { |el| el.is_a?(Numeric) }.inject(:*)
   end
 
   def even_numeric_elements
+    select { |el| el.is_a?(Fixnum) }.select { |num| num.even?}
```

Comment:
Another way to write this would be `select { |el| el.is_a?(Fixnum) && el.even? }`